tests/subsys/pm/power_wakeup_timer/testcase.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+tests:
+  pm.power_wakeup_timer:
+    platform_allow:
+      - stm32l562e_dk
+    tags:
+      - pm
+    harness: power
+    harness_config:
+      pytest_dut_scope: session
+      pytest_args: [--probe=stm_powershield,
+                    --probe-port=<path_to>/STMicroelectronics_PowerShield__Virtual_ComPort]
```

Comment:
These two args (`--probe` and `--probe-port` ) should be provided via the command line, as they vary, as opposed to the `pytest_dut_scope: session` and the properties in `test_expected_values.yaml` file which are "portable"/"fixed" regardless of the probe and port used :)

Reply:
Would it be possible to move
```
-  name: "wakeup_timer"
     measurement_time: 2
     num_of_transitions: 1
     rms: [0.26, 140]
```
to
```
harness_config:
  measurements:
    - name: "wakeup_timer"
      measurement_time: 2
      num_of_transitions: 1
      rms: [0.26, 140]
```
instead of the separate file? not sure what having the commands in a separate file acheives :)